db/migrate/20181107171037_create_postal_addresses.rb

```diff
@@ -0,0 +1,15 @@
+class CreatePostalAddresses < ActiveRecord::Migration[5.1]
+  def change
```

Comment:
Not suuuuper necessary, but it would come in quite handy if you could just squash the two migrations.

For instance: roll back the two migrations, edit this one to have the right names (and delete the other migration) and then run it again.